src/views/Home/components/VictoryPromise/index.js

```diff
@@ -0,0 +1,165 @@
+    this.setState({ CC, IM, LS, LW });
+    let highlight = gordonColors.secondary.green;
+
+    if (CC > 0) {
+      this.setState({ CCColor: highlight });
+    }
+    if (IM > 0) {
+      this.setState({ IMColor: highlight });
+    }
+    if (LS > 0) {
+      this.setState({ LSColor: highlight });
+    }
+    if (LW > 0) {
+      this.setState({ LWColor: highlight });
+    }
+  }
+
+  description() {}
+  render() {
+    /*
+    let daysColor = gordonColors.primary.blue;
+    let swipesColor = gordonColors.secondary.green;
+    let dollarsColor = gordonColors.secondary.yellow;
+    let guestColor = gordonColors.secondary.orange;
+    let emptyColor = gordonColors.neutral.lightGray;
+    */
+    const headerStyle = {
+      backgroundColor: gordonColors.primary.blue,
+      color: '#FFF',
+      padding: '10px',
+      alignItems: 'stretch',
+    };
+
+    const HoverText = withStyles(theme => ({
+      tooltip: {
+        backgroundColor: '#f5f5f9',
+        color: 'rgba(0, 0, 0, 0.87)',
+        maxWidth: 220,
+        fontSize: theme.typography.pxToRem(12),
+        border: '1px solid #dadde9',
+      },
+    }))(Tooltip);
+
+    /*
+    const root = {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'flex-end',
+    };*/
+    //style={{ fontSize: 50, color: this.state.CCColor}}
+    // style={{ fontSize: 50, color: this.state.CCColor}} onMouseOver={() => {console.log('Mouse over'); }}
+    return (
+      <Card /*className="card"*/>
+        <Card /*className="header"*/>
+          <div style={headerStyle}>
+            <Typography style={headerStyle}>VICTORY PROMISE</Typography>
+          </div>
+        </Card>
+        <CardContent className="card-content">
+          <Grid container justify="center">
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Christian Character</Typography>
+                  {"It's very engaging. Right?"}
+                </React.Fragment>
+              }
+            >
+              <HomeIcon
+                className="vpdesign"
+                style={{ fontSize: 50, color: this.state.CCColor }}
+                onClick={this.description()}
+              />
+            </HoverText>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Intellectual Maturity</Typography>
+                  {"It's very engaging. Right?"}
+                </React.Fragment>
+              }
+            >
+              <PeopleIcon
+                className="vpdesign"
+                style={{ fontSize: 50, color: this.state.IMColor }}
+                onClick={this.description()}
+              />
+            </HoverText>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Lives of Service</Typography>
+                  {"It's very engaging. Right?"}
```

Comment:
Do we still want this to say "It's very engaging, right?"

Reply:
Deleted as well

Reply:
I changed the comments in MyProfile Victory promise with the descriptions Chris Carlson recommended